rules/autocorrect-in-outlook

```diff
@@ -0,0 +1,42 @@
+type: rule
+title: Do you know how to use Autocorrect in Outlook to insert common links you often send?
+uri: /rules/autocorrect-in-outlook
```

Comment:
Hi @ae-cher-ssw 

We don't have "/rules/" in the URI

- [ ] 1. Read https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Create-Rules
- [ ] 2. Please fix the URI